### DISCUSSION TITLE
About Configuration file

### DISCUSSION THREAD

**Discussion by zxctw** [streamlink/streamlink#4719] [Q&A] [answered]
As the title, I can't find the Configuration File folder according to the location on the official website.  What should I do to find the Config folder?

**Comment by bastimeyer** [comment, marked as answer]
1. What is it that you don't understand?
2. Which OS are you using?

https://streamlink.github.io/cli/config.html

> On Windows, there is a default config created by the installer, but on any other platform you must create the file yourself.

### Linux

```sh
mkdir -p "${XDG_CONFIG_HOME:-${HOME}/.config}/streamlink/"
"${EDITOR}" "${XDG_CONFIG_HOME:-${HOME}/.config}/streamlink/config"
```

https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html

### macOS

```sh
mkdir -p "${HOME}/Library/Application Support/streamlink/"
open -e "${HOME}/Library/Application Support/streamlink/config"
```

**Reply by zxctw** [reply]
1. So do I need to create a Config folder by myself?
2. openWRT 21.02

**Reply by bastimeyer** [reply]
> On Windows, there is a default config created by the installer, **but on any other platform you must create the file yourself.**

**Reply by zxctw** [reply]
OK, I see！  Thank you！
